tests/backup/backup_basic_test.go

```diff
@@ -62,9 +68,25 @@ var _ = BeforeSuite(func() {
 	log.Infof("Init instance")
 	InitInstance()
 	dash.TestSetBegin(dash.TestSet)
+	// Create the first bucket from the list to be used as generic bucket
+	providers := getProviders()
+	bucket_name := getBucketName()
+	for _, provider := range providers {
```

Comment:
can we add testcasebegin and testcaseend for create and delete buckets, so that these are logged in dashboard

Reply:
Done